packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -72,10 +91,24 @@ export default class QuanticFacet extends LightningElement {
   }
 
   get values() {
-    return this.state.values.map(v => ({
-      ...v,
-      checked: v.state === 'selected'
-    })) || [];
+    return (
+      this.state.values.map((v) => ({
+        ...v,
+        checked: v.state === 'selected',
+        highlightedResult: v.value,
+      })) || []
+    );
+  }
+
+  get query() {
+    return this.input.value;
+  }
+
+  get canShowMoreSearchResults() {
+    if (!this.facet) {
+      return false;
+    }
+    return this.facet.state.facetSearch.moreValuesAvailable;
```

Comment:
```suggestion
    return this.facet && this.facet.state.facetSearch.moreValuesAvailable;
```
If you prefer a shorter version?

Reply:
nicer that way indeed!